sdk/core/Azure.Core/src/ContentType.cs

```diff
@@ -81,7 +81,10 @@ public override int GetHashCode()
         public static bool operator !=(ContentType left, ContentType right)
             => !left.Equals(right);
 
-        /// <inheritdoc />
-        public override string ToString() => _contentType == null?"":_contentType;
+        /// <summary>
+        /// Returns value of <see cref="ContentType"/>.
+        /// </summary>
+        /// <returns>Value of content type <see cref="ContentType"/>.</returns>
```

Comment:
I think we should revert this - the inheritdoc comment is `Returns a string that represents the current object.` which seems accurate and is consistent with other overrides in the repo.

Reply:
Thanks @christothes, I will revert to the old comment.